Library/Formula/mupdf-tools.rb

```diff
@@ -0,0 +1,20 @@
+class MupdfTools < Formula
+  homepage 'http://mupdf.com'
+  url 'http://mupdf.com/downloads/mupdf-1.6-source.tar.gz'
+  sha1 '491d7a3b131589791c7df6dd8161c6bfe41ce74a'
+
+  depends_on :macos => :snow_leopard
+  depends_on "openssl"
+
+  conflicts_with "mupdf",
```

Comment:
I believe the general rule is that if it conflicts with something in a tap we don't bother to mark it as a conflict, but if we do, it'd be something like ` conflicts_with "homebrew/x11/mupdf" `.